sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/impl/JdbcDriverTest.java

```diff
@@ -93,21 +107,39 @@ public void testDriverManager_ddl() throws Exception {
 
   @Test
   public void testInternalConnect_boundedTable() throws Exception {
-    ReadOnlyTableProvider tableProvider =
-        new ReadOnlyTableProvider(
-            "test",
-            ImmutableMap.of(
-                "test",
-                MockedBoundedTable.of(
-                        Schema.FieldType.INT32, "id",
-                        Schema.FieldType.STRING, "name")
-                    .addRows(1, "first")));
-    CalciteConnection connection = JdbcDriver.connect(tableProvider);
+    CalciteConnection connection = JdbcDriver.connect(BOUNDED_TABLE);
     Statement statement = connection.createStatement();
     ResultSet resultSet = statement.executeQuery("SELECT * FROM test");
     assertTrue(resultSet.next());
     assertEquals(1, resultSet.getInt("id"));
     assertEquals("first", resultSet.getString("name"));
     assertFalse(resultSet.next());
   }
+
+  @Test
+  public void testInternalConnect_setDirectRunner() throws Exception {
+    CalciteConnection connection = JdbcDriver.connect(BOUNDED_TABLE);
+    Statement statement = connection.createStatement();
+    assertEquals(0, statement.executeUpdate("SET runner = direct"));
```

Comment:
I suppose we can't test other values until we have a packaging story?

Reply:
A packaging story is part of it. Other runners also require a full integration test as well, they can't just spin up in memory.